scapy/arch/common.py

```diff
@@ -32,18 +31,19 @@ def _check_tcpdump():
     """
     Return True if the tcpdump command can be started
     """
-    with open(os.devnull, 'wb') as devnull:
-        try:
-            proc = subprocess.Popen([conf.prog.tcpdump, "--version"],
-                                    stdout=devnull, stderr=subprocess.STDOUT)
-        except OSError:
-            return False
-
-    if OPENBSD:
-        # 'tcpdump --version' returns 1 on OpenBSD 6.4
-        return proc.wait() == 1
-    else:
-        return proc.wait() == 0
+    try:
+        proc = subprocess.Popen(
+            [conf.prog.tcpdump, "--version"],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT
+        )
+        output = proc.communicate()[0]
+    except OSError:
+        return False
+
+    # tcpdump acts strangely on some OSes and returns 1
+    # therefore we also checks the output
+    return b"tcpdump" in output or proc.returncode == 0
```

Comment:
I would inverse the condition and be more precise on the second case:
```suggestion
    return proc.returncode == 0 or output.startswith(b'Usage: tcpdump ')
```